### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by Romanitho**
# Proposed Changes

- Cleaning things up

### CODE REVIEW

**Comment by Romanitho** [comment]
@FaserF any idea why MegaLinter cannot create the PR ?
image

File: Winget-AutoUpdate-Install.ps1 [hunk 1]
```diff
@@ -231,7 +230,7 @@ function Install-WinGet {
 
         #Install/Update Winget
         Update-WinGet $WinGetAvailableVersion
-
+        Write-ToLog "-> WinGet v$WinGetInstalledVersion installed!`n" "Green"
```
Comment (KnifMelti) [inline Winget-AutoUpdate-Install.ps1:233]: Function `Update-WinGet` already has `Write-ToLog` for all states (also **success**) in it:
```
    $download_string = "-> Downloading WinGet MSIXBundle for App Installer..."
    $install_string = "-> Installing WinGet MSIXBundle for App Installer..."
    $success_string = "-> WinGet MSIXBundle (v$WinGetAvailableVersion) for App Installer installed successfully"
    $reset_string = "-> WinGet sources reset."
    $fail_string = "-> Failed to install WinGet MSIXBundle for App Installer..."

```

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1:233]: Thanks for your review :)

Comment (KnifMelti) [reply Winget-AutoUpdate-Install.ps1]: ...only missing a **CR** if that is of importance?

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1]: ?

Comment (KnifMelti) [reply Winget-AutoUpdate-Install.ps1]: You had:
image

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1]: Put back. Is that good now? :)

Comment (KnifMelti) [reply Winget-AutoUpdate-Install.ps1]: Haha, it's not me that wanted that (I don't se why), I just thought you thought it was important...
...and now you've put it on just two lines (3 missing) 😁:
image

Comment (KnifMelti) [reply Winget-AutoUpdate-Install.ps1]: I'll won't interfera again 😉

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1]: ho, it's just to separate things in display. So in between other string not necessary to add new line. But it's just cosmetic 😅

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1]: trying to clean thing to preprare an upgrade option inside the install script. Your `NoClean` option is a good starting point 😊

File: Winget-AutoUpdate-Install.ps1 [hunk 2]
```diff
@@ -93,7 +93,7 @@ Configure WAU to bypass the Black/White list when run in user context. Applicati
 [CmdletBinding()]
 param(
     [Parameter(Mandatory = $False)] [Alias('S')] [Switch] $Silent = $false,
-    [Parameter(Mandatory = $False)] [Alias('Path')] [String] $WingetUpdatePath = "$env:ProgramData\Winget-AutoUpdate",
+    [Parameter(Mandatory = $False)] [Alias('Path', 'WingetUpdatePath')] [String] $WAUinstallPath = "$env:ProgramData\Winget-AutoUpdate",
```
Comment (Romanitho) [inline Winget-AutoUpdate-Install.ps1:96]: Added alias to keep historical argument name working.

**Comment by Romanitho** [comment]
I think first cleaning part is done

**Comment by FaserF** [comment]
> @FaserF any idea why MegaLinter cannot create the PR ? image

I will have a look into it

File: Winget-AutoUpdate-Install.ps1 [hunk 3]
```diff
@@ -197,47 +197,42 @@ function Install-WingetAutoUpdate {
...
-                if ((Test-Path "$PSScriptRoot\included_apps.txt")) {
-                    Copy-Item -Path "$PSScriptRoot\included_apps.txt" -Destination $WAUinstallPath -Recurse -Force -ErrorAction SilentlyContinue
-                }
-                else {
-                    if (!$ListPath) {
-                        New-Item -Path $WAUinstallPath -Name "included_apps.txt" -ItemType "file" -ErrorAction SilentlyContinue | Out-Null
-                    }
-                }
+            #Else, only copy the "included_apps.txt" list if not existing in WAU
+            elseif (Test-Path "$WAUinstallPath\included_apps.txt") {
```
Comment (KnifMelti) [inline Winget-AutoUpdate-Install.ps1:223]: (!(Test-Path "$WAUinstallPath\included_apps.txt")) maybe.. ..sorry for my curiosity 🤪

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1:223]: You're perfect... my favourite supervisor dinosaur 🦕 😍 
Thanks again for reviewing and pointing out my mistakes.

Comment (Romanitho) [reply Winget-AutoUpdate-Install.ps1]: and now, pure draft at the moment:
https://github.com/Romanitho/Winget-AutoUpdate/tree/draft-upgrade

**Comment by KnifMelti** [comment]
...or better; if it was a success, fail or of it already was the latest one installed.

**Comment by KnifMelti** [comment]
I'm creating a propose branch with returns: success/fail/current

**Comment by FaserF** [comment]
> > @FaserF any idea why MegaLinter cannot create the PR ? image
> 
> I will have a look into it

see here: https://github.com/Romanitho/Winget-AutoUpdate/pull/469

**Pull request merged**